PyInstaller/hooks/hook-boto3.py

```diff
@@ -0,0 +1,8 @@
+from PyInstaller.utils.hooks import collect_data_files, collect_submodules
+
```

Comment:
Please add a comment stating what boto3 is, a link to it's homepage and which version you tested.